README.md

```diff
@@ -209,6 +209,20 @@ with VisierSession(auth) as s:
     print(df_list.head)
 ```
 
+### Snapshot Query
+Snapshot is similar `list` API in terms of the structure and granularity of the query result. A Snapshot query can be thought of as a sequence of `list` queries executed at specified times. The Snapshot result can be augmented with a property named `effectiveDateProperty` to include the time of each snapshot. See the referenced sample JSON query definition for how to do this.
```

Comment:
Snapshot queries retrieve a collection of `list` query-style snapshots taken at the defined intervals. A snapshot query's structure and result granularity is similar to a list query. You can think of a snapshot query as a sequence of list queries executed at specific times. 

You can augment the snapshot result with the `effectiveDateProperty` property to include the time of each snapshot. For more information about using the `effectiveDateProperty` property, see the referenced sample JSON query definition. (@hareskoug can we link to it?)

Reply:
Yes, good call. Updated and link added.